style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  height: 1000px;
+  justify-content: center;
+}
+
+section {
+  position: absolute;
+  height: 718px;
+  width: 375px;
+  border: 1px solid;
+}
```

Comment:
- [ ] [OPTIONAL] I would strongly suggest you to remove the borders around `section `element as it is the requirement of the project to make your design look identical to the [figam design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=kEsxQfAVajKcOTqj-0) template and also because those black borders are clashing with the color contrast of your template, so it's better if you remove them

- [ ] [OPTIONAL] Also I would suggest you to use responsive values instead of absolute values like `px`. This will make your design look much easier to adjust e.g. try implementing and see the improvement

```sh
section {
  padding: 4rem 0 5.5rem 0.2rem;
  border-bottom-left-radius: 7rem;
}
```

Figma design:

<img width="214" alt="Screenshot 2023-03-25 at 9 18 56 AM" src="https://user-images.githubusercontent.com/44798044/227695638-f16ac2df-e909-4b81-8468-c8a083edab95.png">

Your design:

<img width="362" alt="Screenshot 2023-03-25 at 8 43 42 AM" src="https://user-images.githubusercontent.com/44798044/227695617-ae9e5b33-7508-4ef3-8926-2e2f4c37d142.png">